shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/PluginController.java

```diff
@@ -185,36 +185,7 @@ public ShenyuAdminResult enabled(@Valid @RequestBody final BatchCommonDTO batchC
         }
         return ShenyuAdminResult.success(ShenyuResultMessage.ENABLE_SUCCESS);
     }
-    
-    /**
-     * sync plugins.
-     *
-     * @return {@linkplain ShenyuAdminResult}
-     */
-    @PostMapping("/syncPluginAll")
-    @RequiresPermissions("system:plugin:modify")
-    public ShenyuAdminResult syncPluginAll() {
-        boolean success = syncDataService.syncAll(DataEventTypeEnum.REFRESH);
-        if (success) {
```

Comment:
why remove this?

Reply:
Because the plugin will be configured as a template, this syncPluginAll method has been moved to the pluginNamespace controller